.cirrus.yml

```diff
@@ -383,6 +384,12 @@ test_build_cache_images_task:
 
     environment_script: '$SCRIPT_BASE/setup_environment.sh |& ${TIMESTAMP}'
     build_vm_images_script: '$SCRIPT_BASE/build_vm_images.sh |& ${TIMESTAMP}'
+    df_script: '${DFCMD}'
+    journalctl_b_script: 'journalctl -b'
+
+    on_failure:
```

Comment:
I see that Cirrus has an `always` directive. Would it make sense to use it here instead of duplicating?

Reply:
Yes that makes lots of sense, thanks for noticing and sharing.  Though I think perhaps as a topic for a separate PR because nearly every task needs to be updated :confused:  Also I'd like to take your advice of keeping PRs small :grin:

Reply:
https://github.com/containers/libpod/pull/3308